### ISSUE TITLE
MVP List

### ISSUE THREAD

**Issue by gorterd** [JoseMiralles/final-project#1]
Nice work on your design docs, Jose! I can tell you put a lot of though into what problems or questions you might encounter.  So, I opened a different issue for each design doc -- overall, the main revisions just relate to what you've chosen to clone / the design of your clone; I think this comes down to a discrepancy between the desktop website (which is what you need to clone) and I assume the app.  Feel free to drop questions about any of my feedback as a comment, or Slack me! Big apologies for the slowness of my feedback, but do let me know if you have questions or need anything else.

### To Revise:
  - [ ] Your final MVP should be creating a Production README. You can always have bonus MVPs, though.
  - [ ] I think that maybe some of the MVPs you've listed are based on the app / mobile experience. However, you'll be needing to clone just the desktop website. This has a few implications:
    - [ ] You probably can't do live run tracking. Instead, I'd recommend making Routes your first MVP; specifically, the Create Route page, with the ability to click on points on the map to generate a route. The create route page, along with the corresponding show and edit pages, would be a hefty first MVP, but a good foundation to build on.
    - If you want to do the user profile, do you intend to do the calendar view, the stats view, or both? Regardless, I'd recommend *first* implementing activity CRUD (including especially the ability to add a new workout with a title, date, time, distance, duration etc, and to tie it to a particular route), which would be the data you'd show on the user profile calendar / from which you'd derive those stats
      * Regardless, I think the user profile and activity CRUD go hand in hand, and I think it would make sense to treat those as 2 MVPs. That would leave one other non-bonus MVP.
    - You can do friends second as well. Make sure to think through how you'd implement finding friends (search?), and accepting friend requests (where do the requests show up?)
    - The activity feed and comments would also have *both* activities and friends as prerequisites. So in terms of non-bonus MVPs, you probably want to either implement the activity feed *or* the user dashboard / stats *or* comments as your final MVP, with friends and activities being your second and third MVPs, in either order. Let me know if you have other thoughts as well!

### Suggestions / Thoughts:
  - Let me know if you intend to use Google Maps (my strong recommendation), or something else, for creating and viewing routes!

**Comment by JoseMiralles** [comment]
@gorterd Thank you so much for all of the feedback.

I didn't realize that MapMyRun had a route drawing website, and I thought that I had to create a run tracking web app instead.

I'm going to re-design my docs to create a route drawing webapp instead. Which is great, because after my cohort is over, I will then have an excuse to use React Native to create a companion mobile run tracking app.

**Comment by gorterd** [comment]
@JoseMiralles Yeah, sorry about that misunderstanding. But I think that's an awesome plan to create a companion mobile app during the job search; they'll look great together in your portfolio!
